contracts/marketplace/CruzoMarket.sol

```diff
@@ -66,14 +73,22 @@ contract CruzoMarket is
     // Service fee percentage in basis point (100bp = 1%)
     uint16 public serviceFee;
 
+    string private _rawVaultFuncSignature;
```

Comment:
Naming: should not have leading underscore

Reply:
U mean __rawVaultFuncSignature?

Reply:
`rawVaultFuncSignature`